pkg/network/dhcp/dhcp.go

```diff
@@ -38,6 +40,45 @@ func (d *Probe) Start(inf string) error {
 	return nil
 }
 
+// Run runs the DHCP client process and registers it to zinit
+func (d *Probe) Run(inf string) error {
+	z, err := zinit.New("")
+	if err != nil {
+		log.Error().Err(err).Msg("failed to connect to zinit")
+		return err
+	}
+
+	serviceName := fmt.Sprintf("dhcp-%s", inf)
+
+	ns, err := exec.Command("ip", "netns", "identify").Output()
+	if err != nil {
+		return errors.Wrap(err, "failed to identify namespace")
+	}
+
```

Comment:
As we agreed, you need to check if another service with the same name already exists! And may be return some error (that can be checked later)

We need also to provide a way to stop that backgroup dhcp process, or check if it exists.

May be it's better if we provide 2 Prope types. The current one can be the default, but a new BackgroundPrope type can also be added that provide 
```go 
NewBackgroundPrope() *BackgroundPrope
Start(interface) error
IsRunning(interface) (bool, error)
// and
Stop(interface) error
```